src/transformers/benchmark/benchmark_utils.py

```diff
@@ -94,7 +94,7 @@ def wrapper_func(queue: Queue, *args):
         return result
 
     if do_multi_processing:
-        logging.info("fFunction {func} is executed in its own process...")
+        logger.info("fFunction {func} is executed in its own process...")
```

Comment:
```suggestion
        logger.info(f"Function {func} is executed in its own process...")
```
(unrelated to the PR but it caught my eye)